src/App.tsx

```diff
@@ -17,18 +75,31 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                handleSelectChange={handleSelectChange}
+                query={query}
+                handleQueryChange={handleQueryChange}
+                clearQuery={clearQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && <Loader />}
+
+              {!isLoading && todos.length > 0 && (
+                <TodoList
+                  todo={filteredTodos}
```

Comment:
The prop 'todo' should be named 'todos' to better reflect that it is an array of todos, following the naming conventions for props.